### PR TITLE
Improve Documentation

### PR DESCRIPTION
**Pull request by ylow**
- Rewrote and simplified up a lot of documentation. 
 - Added API docs
 - Removed extra copy of documentation in pyxet/docs
 - Shifted the starter titanic repo to xethub
 - Fix pypi docs
 - Pull the doc version from the package version

Recommended to checkout and run `make html` in the docs/ folder to see how it looks.

### CODE REVIEW

File: docs/markdowns/mount.md [hunk 1]
```diff
@@ -1,61 +1,54 @@
...
-A key tool for simplifying working with data and CICD.   
-This let you use any tool that works with local files, but with data stored in XetHub.   
-Perfect to explore files like images and text, saving monitoring logs, and dump databases data for easy recovery.
+PyXet allows you to perform a filesystem mount of any version of any Xet repository
+allowing you to immediately access TBs of data anywhere without having to download 
+everything.
 
-```python
-import pyxet
+This let you use any tool that works with local files, but with data stored in
```
Comment (port8080) [inline docs/markdowns/mount.md:7]:
```suggestion
This lets you use any tool that works with local files, but with data stored in
```

File: docs/markdowns/mount.md [hunk 2]
```diff
@@ -1,61 +1,54 @@
...
+XetHub.  This is perfect for exploring larger images and text files, as well as 
+Sqlite and Parquet databases.
```
Comment (port8080) [inline docs/markdowns/mount.md:9]:
```suggestion
SQLite and Parquet databases.
```

File: docs/markdowns/quickstart.md [hunk 3]
```diff
@@ -20,6 +18,38 @@ Then, install pyxet with:
 $ pip install pyxet
 ```
 
+(accountsetup)=
```
Comment (port8080) [inline docs/markdowns/quickstart.md:21]: Not sure, but this `(accountsetup)=` seems to render for me and that does not seem to be your intent

File: docs/markdowns/quickstart.md [hunk 4]
```diff
@@ -20,6 +18,38 @@ Then, install pyxet with:
...
+## Authentication
+
+Signup on [XetHub](https://xethub.com/user/sign_up) and obtain
+a username and access token. You should write this down.
```
Comment (port8080) [inline docs/markdowns/quickstart.md:25]:
```suggestion
a username and personal access token. You should write this down.
```

**Review by port8080 — APPROVED** [review]
:shipit: 🚀

File: docs/markdowns/quickstart.md (hunk 3 above)
Comment (ylow) [reply docs/markdowns/quickstart.md:21]: oh? It doesn't render for me. Its supposed to create a explicit crossreference target though I am not able to figure out how to actually use it...

**Pull request merged**